service-front/module/Application/view/application/authenticated/lpa/status/index.twig

```diff
@@ -69,35 +69,31 @@
                         <p>We’re currently asking customers to allow 30 working days for us to receive the LPA, enter it on to our system and update this message.</p>
                 {% endif %}
 
-                <p>If you think we should have received your LPA by now, please call 0300 456 0300</p>
-                <p>Opening times: Monday, Tuesday, Thursday, Friday 9.30am to 5pm. Wednesday 10am to 5pm</p>
+                <p>If you think we should have received your LPA by now, please call 0300 456 0300.</p>
+                <p>Opening times: Monday, Tuesday, Thursday, Friday 9.30am to 5pm. Wednesday 10am to 5pm.</p>
             </div>
 
-
-        {% elseif status == 'returned' %}
+        {% elseif status == 'processed' %}
 
             <div>
-                <h2 class="heading-xlarge">We’ve returned the LPA</h2>
+                <h2 class="heading-xlarge">We’ve processed the LPA</h2>
             </div>
+
             <div>
-                <p>We've sent the LPA to {{ lpa.document.correspondent.name.title }} {{ lpa.document.correspondent.name.first }} {{ lpa.document.correspondent.name.last }}.</p>
-                <p>For security reasons, we can only tell you we’ve returned the LPA. We cannot tell you on this service if the LPA is registered or not.</p>
-                <p>If you’re the donor or an attorney, you’ll receive a letter that tells you the LPA is:</p>
+                <p>For security reasons, we cannot tell you here if the LPA has been registered or not.</p>
+
+                <p>{{ lpa.document.correspondent.name.title }} {{ lpa.document.correspondent.name.first }} {{ lpa.document.correspondent.name.last }} will receive the LPA in the post, along with a letter saying if the LPA:
+
                 <ul class="list list-bullet" style="font-weight:normal;">
-                    <li>registered</li>
-                    <li>unregistered - we’ll also say why we could not register it and what to do next</li>
+                    <li>has been registered</li>
+                    <li>could not be registered - we’ll also say why we could not register it and what to do next</li>
                 </ul>
 
-                {% if returnDate != null  %}
-                    <p>We sent the LPA and letters by second class post on  {{ returnDate|date("d F Y")  }}. </p>
-                {% else %}
-                    <p>We sent the LPA and letters by second class post. </p>
-                {% endif %}
+                <p>The donor and all attorneys on the LPA will get a letter telling them the outcome.</p>
 
-                <p>If they have not arrived after 10 working days, please call 0300 456 0300.</p>
+                <p>If the LPA or the letters have not arrived by <span data-cy="lpa-should-receive-by-date">{{ shouldReceiveByDate|date('d/m/y') }}</span>, please call 0300 456 0300.</p>
```

Comment:
`shouldReceiveByDate` could be null

Reply:
If `shouldReceiveByDate` is null, the template handles it without an error, showing today's date. Given that it's highly unlikely a user could reach this page for an LPA with no dates set on it at all (which would result in a null processed date and therefore show today's date), this is a risk I'm willing to accept.

Reply:
I wouldn't assume so, worth checking. But I would assume this is populated when `received` is populated. As I think the logic should be once we have a received date we then populate `shouldReceiveByDate` with our current wait time info, which I think is 8 weeks?